.devcontainer/host-metering.conf

```diff
@@ -0,0 +1,17 @@
+# Test host-metering configuration file which writes to a local prometheus
+# and uses a test certificate and key.
+
+# E.g. could be used as:
+# ./host-metering -config hack/host-metering.conf daemon
+
+[host-metering]
+write_url=http://prometheus:9090/api/v1/write
```

Comment:
To improve part of the performance degradation. The `host-metering.conf` could use `localhost` and only in the "compose" envs override it with `HOST_METERING_WRITE_URL`, e.g by setting this env var in `.vscode/launch.json` `env` var part or in the `docker-compose.yml` in `environment` for `host-metering` service.  This would also mean to not use the compose for  `make test-daemon`.